gamemaster.py

```diff
@@ -44,6 +50,7 @@ def update(self, events):
             if self.basic_grunt_spawn_delay_ < self.minimum_spawn_delay_:
                 self.basic_grunt_spawn_delay_ = self.minimum_spawn_delay_
         if self.basic_grunt_spawn_timer_ > self.basic_grunt_spawn_delay_:
+            self.spawn_not_so_basic_grunt()
```

Comment:
I don't like how they're being spawned together, but I don't have any easy solutions either. It'll be messy to add another spawn timer (might be okay for prototype), if we do that the correct solution would be create a spawner class.